x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -5,51 +5,48 @@
  * 2.0.
  */
 
-import React from 'react';
 import { merge } from 'lodash';
+import React from 'react';
 import { euiDarkVars } from '@kbn/ui-shared-deps-src/theme';
 import { I18nProvider } from '@kbn/i18n-react';
 import { ThemeProvider } from 'styled-components';
 import { CasesContextValue, DEFAULT_FEATURES, SECURITY_SOLUTION_OWNER } from '../../../common';
 import { CasesProvider } from '../../components/cases_context';
 import { createKibanaContextProviderMock } from '../lib/kibana/kibana_react.mock';
 import { FieldHook } from '../shared_imports';
-
 interface Props {
   children: React.ReactNode;
-  userCanCrud?: boolean;
-  features?: CasesContextValue['features'];
+  caseConfig?: Partial<CasesContextValue>;
```

Comment:
I think is best to be specific in the props instead of using an object as props are already an object of values. The reason is that a) objects cause unnecessary re-renders even for the same values if it is not handled correctly (I know it is not important for tests) b) it is more readable c) A consumer cannot do <TestProviders userCanCrud /> with objects. An object would make sense if you wanted to provide configuration for each of the providers that are children of the  `TestProviders` component. For example: `contextConfig, i18nConfig, themeConfig` etc

You can do:

`type Props = { children: React.ReactNode; } & Partial<CasesContextValue>`

Reply:
got it, thanks!